docs/docs/user/references/config.md

```diff
@@ -1 +1,15 @@
 # Configuration
+
+KPOps reads most of a [pipeline](/user/getting-started/quick-start/#deploying-the-word-count-pipeline)'s configuration that is not related to its [components](./components.md) from [`config.yaml`](/user/references/config).
```

Comment:
```suggestion
KPOps reads most of a [pipeline](/user/getting-started/quick-start/#deploying-the-word-count-pipeline)'s configuration that is not related to its [components](./components.md) from [`config.yaml`](/user/references/config.md).
```
necessary?

Reply:
I suppose at least a little explanation on what the file is used for could be beneficial for a complete newbie?

Reply:
I was referring to the `.md` file ending because it is used for the other link

Reply:
Ah, no it's not needed. I opted for removing it from the other one as well.